src/v/io/pager.cc

```diff
@@ -181,7 +181,8 @@ pager::get_page(uint64_t offset) noexcept {
             if (page->test_flag(page::flags::faulting)) {
                 page::waiter waiter;
                 page->add_waiter(waiter);
-                co_await waiter.ready.get_future();
```

Comment:
Can't this be one line. I.e,
```
co_await std::move(waiter.ready.get_future());
```

Reply:
maybe. i didn't try this one because the temporary should bind to the same thing that the rvalue of std::move would produce and figured clang-tidy would still have a problem with it.